--- a/Makefile.in	2026-02-27 20:56:00.726938124 -0500
+++ b/Makefile.in	2026-02-27 20:56:00.740951087 -0500
@@ -1149,6 +1149,12 @@
 
 install-data-am: install-aclocalDATA install-pkgconfigDATA
 
+install-data-hook:
+	$(MKDIR_P) $(DESTDIR)/usr/lib/pkgconfig
+	test -f xorg-server.pc && $(INSTALL_DATA) xorg-server.pc $(DESTDIR)/usr/lib/pkgconfig/xorg-server.pc || true
+
+install-data-am: install-data-hook
+
 install-dvi: install-dvi-recursive
 
 install-dvi-am:
